app/controllers/admin/groups_controller.rb

```diff
@@ -0,0 +1,63 @@
+class Admin::GroupsController < ApplicationController
```

Comment:
You are totally right. All admin controllers should inherit AdminController. This way I have duplicated stuff in AdminController and Admin::GroupsController.

> Maybe in another PR, I don't mind, just asking what you think about this. :-)

I wanted to do it in next PR, do you think it would be more proper doing it in this one?